frontend/src/packages/journal/journal-api.ts

```diff
@@ -48,6 +49,22 @@ class JournalApi extends BaseHttpApi {
 
     return await response.json<JournalEntryGetAllItemResponseDto>();
   }
+
+  public async deleteJournalEntry(
+    id: number,
+  ): Promise<JournalEntryDeleteResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(JournalApiPath.ROOT, `${id}`, {}),
```

Comment:
Create another item for id

Reply:
@v0ldemar01 what do you mean by this, like:
    const idItem = String(id);
const response = await this.load(
      this.getFullEndpoint(JournalApiPath.ROOT,  idItem, {}),

Reply:
I think @v0ldemar01 means that you need to add `$ID: '/:id',` to JournalApiPath and use it here.